cmake/CMakeLists.txt

```diff
@@ -255,8 +262,8 @@ if(PKG_MSCG OR PKG_USER-ATC OR PKG_USER-AWPMD OR PKG_USER-QUIP OR PKG_LATTE)
     enable_language(Fortran)
     file(GLOB LAPACK_SOURCES ${LAMMPS_LIB_SOURCE_DIR}/linalg/[^.]*.[fF])
     add_library(linalg STATIC ${LAPACK_SOURCES})
-    set(BLAS_LIBRARIES linalg)
-    set(LAPACK_LIBRARIES linalg)
+    set(BLAS_LIBRARIES "${CMAKE_BINARY_DIR}/liblinalg.a")
+    set(LAPACK_LIBRARIES "${CMAKE_BINARY_DIR}/liblinalg.a")
```

Comment:
That feel wrong...why is that needed?

Reply:
because LATTE won't find it otherwise and thus configuration of LATTE fails unless there is a publicly installed BLAS/LAPACK library.

Reply:
On a second though that is ok, just add a comment.

Reply:
we might still want to use:
```
get_target_property(BLAS_LIBRARIES linalg LOCATION)
get_target_property(LAPACK_LIBRARIES linalg LOCATION)
```
instead.

Reply:
Even though merged we should still do this.

Reply:
Please figure it out and submit a pull request. I have googled until my fingers started to bleed, but I either get an error message from cmake telling me that I should not to this, or did not get the answer I am looking for. This works well enough and I need the time to complete other open PRs by tonight.

Reply:
CMake's error message basically told me what to do: see #1950.